.github/workflows/commit.yaml

```diff
@@ -14,7 +14,7 @@ on:
       - 'netlify.toml'
 
 env:  # Update this prior to requiring a higher minor version in go.mod
-  GO_VERSION: "1.23"
+  GO_VERSION: "1.22"
```

Comment:
Upgrading this is breaking `goanalysis_metalinter`:
https://github.com/tetratelabs/wazero/actions/runs/10378334638/job/28734383081?pr=2301#step:7:190

Reply:
Still failing with `golangci-lint@v1.59.1`.